### ISSUE TITLE
issues with samtools mpilup

### ISSUE THREAD

**Issue by akinyanju** [lh3/psmc#8]
When I ran this command for the psmc

samtools mpileup -C50 -uf fasta.fa recal.bam | bcftools call -c - | vcfutils.pl vcf2fq -d 10 -D 100 | gzip > out.fq.gz,

I got below message hanging on the screen and output file "out.fq.gz" size remains at 0. I tried also setting -d to default and again to 8000 but still getting same error. I am wondering if i am doing anything typically wrong

[mpileup] 1 samples in 1 input files
<mpileup> Set max per-file depth to 8000

Thank you.
Lawal

**Comment by lh3** [comment]
You have to use samtools-0.1.18 or samtools-0.1.19. The latest samtools won't work.

**Comment by akinyanju** [comment]
Hi Li,

Thank you for your response. I used the version of samtools as suggested and never get the previous error any longer. However, I started getting another error with the bcftools (see below). I used bcftools 1.3 but when I observed there could be incompatibility with the version of samtools, I then installed old version 1.1. the error still persisted.

vcf.c:753 bcf_hdr_read] invalid BCF2 magic string: only BCFv2.2 is supported.
Failed to open: -
Use of uninitialized value $l in numeric lt (<) at vcfutils.pl line 566.
Use of uninitialized value $l in numeric lt (<) at vcfutils.pl line 566.
